i18n-helpers/src/preprocessors.rs

```diff
@@ -0,0 +1,2 @@
+pub mod gettext;
+pub use gettext::Gettext;
```

Comment:
I'm not a fan of giving multiple public names for the same thing — could you instead make the module private?

```suggestion
mod gettext;
pub use gettext::Gettext;
```

Reply:
I've fixed it.